modules/ROOT/pages/osv2-tutorial.adoc

```diff
@@ -1,335 +1,129 @@
-+
-.The screenshot shows (*1*) the vertical ellipsis menu and (*2*) the *Rename* option.
-+
-image::os-tutorial-dc-rename-flow.png[Vertical ellipsis menu and Rename option]
-
-. Click the plus sign to the right of *Flows*.
-. In the wizard, click *Go straight to canvas*:
-+
-.The arrow shows the *Go straight to canvas* option.
-image::os-tutorial-dc-goto-canvas.png[Go straight to canvas option]
-+
-. Click the *Trigger* box and enter `H` to search for HTTP Listener:
-+
-.The arrow shows the HTTP Listener.  
-image::os-tutorial-dc-http-search.png[Search for HTTP Listener]
-+
-. Click *HTTP Listener*.
-. In the *HTTP Listener* window, set the value of *Path* to `/retrieve` and then click the close button (*X*):
-+
-.The screenshot shows (*1*) the *Path* value and (*2*) the close button (*X*).
-image::os-tutorial-dc-http-retrieve-path.png[Path value and close button (X)]
-+
-Because the default settings for HTTP Listener are correct, you don't need to click *Edit*.
-. Rename this new flow to `Retrieve`.
-. Click the plus icon (+) to the right of the HTTP Listener trigger to add Object Store Connector.
-. In the *Select a component* window, enter `ob` to search for ObjectStore Connector:
-+
-.The arrow shows ObjectStore Connector.
-image::os-tutorial-dc-os-search.png[Search for Object Store Connector]
-+
-. Click the connector and then click the *Retrieve* operation.
-. In the *Configuration* tab, specify `#[attributes.queryParams.key]` in the *Key* field:
-+
-.The arrow shows the *Key* field in the *Retrieve* configuration.
-image::os-tutorial-dc-retrieve-values.png[Key field for the Retrieve operation]
-
-. Click *Test* to run the app in a temporary location.
-. Click the copy icon to copy the deployment URL:
-+
-.The arrow shows the deployment URL copy icon.
-image::os-tutorial-dc-copy-link.png[The copy deployment URL icon]
-. From the command line, run the following `curl` command to retrieve the key, replacing _DEPLOYMENT-URL_ with the deployment URL that you copied from Design Center:
-+
-[source,console]
-----
-curl http://DEPLOYMENT-URL/retrieve?key=myKey;echo
-----
-+
-The output from this command displays the value for the `myKey`:
-+
-[source,console]
-----
-"OSv2 Test"
-----
-
-
-You have successfully tested your app using a `curl` command to submit a REST API command to the Object Store API and to retrieve the results.
-
-Next, you create a Mule app in Anypoint Studio 7 and add an object store.
-
-
-== Create a Mule 4 App with an Object Store in Anypoint Studio 7
-
-You can now use Anypoint Studio 7 to create a Mule 4 app similar to the app you created in Design Center.
-
-First, you use Studio 7 to configure an app that includes an HTTP Listener trigger and Object Store Connector.
+First, use Studio 7.x to configure an app that includes an HTTP Listener trigger and Object Store Connector.
 Then, you deploy the app to CloudHub and use a `curl` command to send a key and value, and view the key in Runtime Manager.
```

Comment:
Remove the "you" for consistency with the previous paragraph. Also, the sentence can be improved (I know that you probably didn't write this.)

Then, deploy the app to CloudHub, use a `curl` command to send a key and value, and view the key in Runtime Manager.